src/Page.php

```diff
@@ -776,7 +776,7 @@ public function keyboard()
     }
 
     /**
-     * @return Dom
+     * Get Dom object
```

Comment:
just delete this doc

Reply:
Do you want me to delete "Get Dom object" too?

Reply:
yes, doesn't add any value. the function is called dom and typed with `Dom` :trollface: